### PR TITLE
Feature/retention policy

### PR DESCRIPTION
**Pull request by marktaling**

### CODE REVIEW

File: src/Augurk.Api/Managers/ConfigurationManager.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            Configuration configuration = null;
+
+            using (var session = Database.DocumentStore.OpenAsyncSession())
+            {
+                configuration = await session.LoadAsync<Configuration>(KEY);
+            }
+
+            if (configuration == null)
+            {
+                configuration = new Configuration
```
Comment (jmezach) [inline src/Augurk.Api/Managers/ConfigurationManager.cs]: Perhaps a comment here would be nice to indicate that these are the defaults. Would make finding the defaults back in the future easier.

File: src/Augurk.Api/Managers/CustomizationManager.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            Customization customizationSettings = null;
+
+            using (var session = Database.DocumentStore.OpenAsyncSession())
+            {
+                customizationSettings = await session.LoadAsync<Customization>(KEY);
+            }
+
+            if (customizationSettings == null)
+            {
+                customizationSettings = new Customization
```
Comment (jmezach) [inline src/Augurk.Api/Managers/CustomizationManager.cs]: Comment indicating these are the defaults

File: src/Augurk.Specifications/Gherkin/NotImplementedTag.feature.cs [hunk 3]
```diff
@@ -1,22 +1,21 @@
 ﻿// ------------------------------------------------------------------------------
 //  <auto-generated>
 //      This code was generated by SpecFlow (http://www.specflow.org/).
-//      SpecFlow Version:1.9.0.77
-//      SpecFlow Generator Version:1.9.0.0
-//      Runtime Version:4.0.30319.42000
+//      SpecFlow Version:2.1.0.0
```
Comment (jmezach) [inline src/Augurk.Specifications/Gherkin/NotImplementedTag.feature.cs]: Perhaps we should remove the *.feature.cs files and add them to the .gitignore.

File: src/Augurk/templates/configuration.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<div>
+    <h2>Customization & Configuration</h2>
```
Comment (jmezach) [inline src/Augurk/templates/configuration.html]: Shouldn't this be &amp; ?

File: src/Augurk/Scripts/angular-bootstrap-toggle.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
+(function () {
```
Comment (jmezach) [inline src/Augurk/Scripts/angular-bootstrap-toggle.js]: Angular 1 feels so dirty to me these days ;).

Comment (marktaling) [reply src/Augurk/Scripts/angular-bootstrap-toggle.js]: Deal with it.

File: src/Augurk.Api/Managers/ConfigurationManager.cs (hunk 1 above)
Comment (marktaling) [reply src/Augurk.Api/Managers/ConfigurationManager.cs]: Fair enough. Will move them to Defaults.cs

File: src/Augurk.Api/Managers/CustomizationManager.cs (hunk 2 above)
Comment (marktaling) [reply src/Augurk.Api/Managers/CustomizationManager.cs]: Fair enough. Will move them to Defaults.cs

File: src/Augurk/templates/configuration.html (hunk 4 above)
Comment (marktaling) [reply src/Augurk/templates/configuration.html]: Amp-ed it up 👍

**Pull request merged**